website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -228,6 +228,46 @@ google_cloud_kms_config {
   key_version_resource_id = ""
 }
 ```
+## Migration from Cloud Provider Snapshot Backup Policy to Cloud Provider Backup Schedule
```

Comment:
Still would like some input here @coderGo93 and @thetonymaster

Reply:
Hey @themantissa I've been thinking about this the last past couple of days, this could be a simple tool that translates from one to other, although don't know how simple this really could be...

Reply:
Either way they could just import it if there's already one that works for them...

Reply:
So the short answer is, it's complicated to do this for them?  That's fair.  I think in general if we can do anything to make the transition from deprecated resource to new easy than we should.   In this case the rewrite is enough that I think it's worth it for users to manually do it and less likely to have issues.